variables.tf

```diff
@@ -246,6 +246,12 @@ variable "policy" {
   default     = null
 }
 
+variable "policy_path" {
+  description = "Path of IAM policies to use for Step Function"
+  type        = string
+  default     = "/"
```

Comment:
Default to null is better. Please update.

Reply:
Default is updated to null.